src/lib/supabaseClient.ts

```diff
@@ -0,0 +1,6 @@
+import { createClient } from '@supabase/supabase-js';
+
+const supabaseUrl = import.meta.env.VITE_SUPABASE_URL;
+const supabaseKey = import.meta.env.VITE_SUPABASE_ANON_KEY;
```

Comment:
Czasami warto zrezygnować z dodatkowego powielania kontekstu w nazwach zmiennych. To ze są to zmienne zwiazane z supabase widać po nazwie enva, oraz po nazwie pliku, wiec prefix "supabase" jest raczej zbedny.